src/main/java/storage/StorageFileException.java

```diff
@@ -0,0 +1,4 @@
+package storage;
+
+public class StorageFileException extends Exception{
+}
```

Comment:
Glad to see you intend to create storage file exceptions! Bodes well for our error handling 😄